extensions/wikidata/tests/src/org/openrefine/wikidata/qa/MockConstraintFetcher.java

```diff
@@ -66,6 +69,11 @@
     public static QuantityValue minValuePid = Datamodel.makeQuantityValue(new BigDecimal(0));
     public static QuantityValue maxValuePid = Datamodel.makeQuantityValue(new BigDecimal(150));
 
+    public static PropertyIdValue conflictsWithPid = Datamodel.makeWikidataPropertyIdValue("P50");
+    public static PropertyIdValue pidConflictingStatement = Datamodel.makeWikidataPropertyIdValue("P31");
+    public static Value conflictingStatementValue = Datamodel.makeWikidataItemIdValue("Q5");
+    public static Value conflictsWithValue = Datamodel.makeWikidataItemIdValue("Q36322");
```

Comment:
I recognize Q5, but the rest would have to look up. For the benefit of future readers, it might be nice to give them mnemonic variable names or to put the string values in constants.